### PR TITLE
Move control button to the subtitle bar

### PR DESCRIPTION
**Pull request by saranshdhingra**
The PR adds 3 small changes:
- Moves the control button to the subtitle bar as per new design
- Fixes the table's header color as per the `theme primary`.
- Re-adds the logo(somehow missed in the last PR).

Here is the latest view as per this PR:
image

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -22,7 +22,9 @@
     "prettier": "2.4.1"
   },
   "dependencies": {
+    "@material/button": "^13.0.0",
```
Comment (enocom) [inline web/package.json:25]: Should we be locking dependencies to a version for deterministic builds?

**Review by enocom — DISMISSED** [review]

**Comment by enocom** [comment]
We'll need to verify this looks good on mobile, but that can wait for the final PR in this series and once everything looks good we'll deploy it.

**Review by enocom — CHANGES_REQUESTED** [review]
Let’s lock to a specific version of dependencies. We can install a bit to upgrade these later.

**Comment by saranshdhingra** [comment]
> Let’s lock to a specific version of dependencies. We can install a bit to upgrade these later.

Sure thing. Haven't fixed the parcel dependencies, though. I thought, since they are devDependencies, we can be a bit flexible there. Let me know if you want to fix them too, we can do it in the next PR :)

**Comment by enocom** [comment]
Let's just lock everywhere. Since parcel is building for us, I'm not sure if we have an production dependencies strictly speaking.

**Pull request merged**